src/evaluations/data/tests/evaluation_configs_test.py

```diff
@@ -29,43 +29,77 @@ class EvaluationConfigTest(absltest.TestCase):
       'wfa_cardinality_estimation_evaluation_framework.evaluations.data.'
       + 'evaluation_configs.')
 
-  def test_generate_configs_scenario_3b_set_sizes_correct(self):
-    conf_list = evaluation_configs._generate_configs_scenario_3b(
-        universe_size=200,
-        num_sets=3,
-        small_set_size=50,
-        large_set_size=100,
-        user_activity_assciation=(set_generator
-                                  .USER_ACTIVITY_ASSOCIATION_IDENTICAL)
-    )
+  def test_generate_configs_scenario_1_2_set_sizes_correct(self):
+    for remarketing_rate in [None, 0.2]:
```

Comment:
It'd be better to use a parameterized test here. Makes test debugging easier... less whack-a-mole type debugging.

Reply:
Was following the existing code logic, let me refactor a little bit